src/services/provider/layerInfoResult.provider.js

```diff
@@ -0,0 +1,27 @@
+import { $injector } from '../../injection';
+import { LayerInfoResult } from '../../modules/layerInfo/services/LayerInfoService';
+
+/**
+ * Uses the BVV endpoint to load layerinfoResult.
+ * @function
+ * @returns {Promise<LayerInfoResult>}
+ */
+export const loadBvvLayerInfo = async (geoResourceId) => {
+
+	const { HttpService: httpService, ConfigService: configService } = $injector.inject('HttpService', 'ConfigService');
+	const url = `${configService.getValueAsPath('BACKEND_URL')}layerinfo/${geoResourceId}`;
+
+	const result = await httpService.get(url);
+
+	switch (result.status) {
+		case 200: {
+			const htmlContent = await result.text();
+			return new LayerInfoResult(htmlContent);
+		}
+		case 404: {
+			return null;
+		}
+	}
+
+	throw new Error(`LayerInfoResult for '${geoResourceId}' could not be loaded`);
+};
```

Comment:
File must be located in `src/modules/layerInfo/services/provider`.
Same for the corresponding test file.